docs/google/walkthrough.md

```diff
@@ -0,0 +1,83 @@
+# Setting up Kubernetes 1.7 in Google with TLS secured private VPN mesh
+
+In the following, we're going to show you how to use `kubicorn` to ramp up a Kubernetes cluster in Google cloud, use it and tear it down again.
+The cluster will be running over Google private networking on an encrypted VPN mesh.
+
+As a prerequisite, you need to have `kubicorn` installed. Since we don't have binary releases yet, we assume you've got Go installed and simply do:
+
+#### Installing
+
+```
+$ go get github.com/kris-nova/kubicorn
+```
+
+The first thing you will do now is to define the cluster resources.
+For this, you need to select a certain profile. Of course, once you're more familiar with `kubicorn`, you can go ahead and extend existing profiles or create new ones.
+In the following we'll be using an existing profile called `do`, which is a profile for a cluster in Google.
+
+#### Creating
+
+You will need to create a project in you google cloud account.
+This project will get a projectid, something like kubicorn-132742.
+When using Google Cloud Compute Egnine the Kubicorn name and the project ID need to be the same value.
```

Comment:
This seems unnecessarily prohibitive. I'd expect the kubicorn cluster name and GCP project ID to be completely orthogonal. I'd rather see the project modeled in the cluster definition, especially for cases where admins want to run their masters in one project and nodes in another project (some consider this a security best practice). Some also want to run multiple node pools in different projects for better billing granularity.